src/Symfony/Component/HttpKernel/Kernel.php

```diff
@@ -511,17 +511,18 @@ protected function getContainerBaseClass()
     protected function initializeContainer()
     {
         $class = $this->getContainerClass();
-        $cache = new ConfigCache($this->getCacheDir().'/'.$class.'.php', $this->debug);
-        $fresh = true;
-        if (!$cache->isFresh()) {
-            $container = $this->buildContainer();
-            $container->compile();
-            $this->dumpContainer($cache, $container, $class, $this->getContainerBaseClass());
 
-            $fresh = false;
-        }
+        if (!$fresh = class_exists($class, false)) {
```

Comment:
Must be a bug with require_once, the file is included twice otherwise. I think it doesn't do any harm so we should merge. (Yes I know...)